src/CloneLeeroy/CloneLeeroy.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFrameworks>net5.0;net6.0</TargetFrameworks>
```

Comment:
Not sure we need to keep `net5.0`, since it's EOL in May: https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core

But no strong objection to dual-targeting.